CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 I2S library change log
 ======================
 
+3.0.1
+-----
+
+  * CHANGE: Configure LR clock for input at initialisation.
```

Comment:
It might be worth noting that the change only applies to I2S slave mode, such that master mode users know they do not need to upgrade here.

Reply:
Change log entry expanded as you suggested.